general/OKRs/README.md

```diff
@@ -0,0 +1,164 @@
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+* Launch of deduplication of code
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+* Decide on the faith of legacy projects
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+
+\* we're lost on the name still, this objective needs its own document to explain better since it's a new product
+
+### Developer Community:
+
+* Create a network of influencers incl. Developers, Companies, Academia
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+* Increase our presence in the developer community in Madrid & SF
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+* Host and attend conferences and meetups
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+* Keep track of who you met that was relevant to source{d}/relevant new topics in the MLoSC community
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+* Publish blogposts at a regular pace
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+* Start cross posting to Medium
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+* Start publishing videos
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+* Publish a page where we show all source{d} related videos
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+* Redefine how we use social media and community Slack
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+
+### Academia:
+
+* Getting academia to use our tech
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+* Understand usecases
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+* Understand workflow
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+* Gather inputs on what 'ImageNet for Code' is to stakeholders in the community
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+* Deepen our relationship with universities
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+
+### Research
+
+* Write scientific papers
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+* Research structural embeddings
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+* Explore new directions and emerging papers
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+
+### Engineering:
+
+* Collect, store and process 10s of millions of repositories using the pipeline in Paris
+  * Improve the borges performance to be able to download all the repositories in 1 month with 60 machines.
+  * Scale our infrastructure to support this - a cluster of 60 machines.
+  * Develop a system to keep this updated with a delay of 1 week.
+* Launch a production level engine
+  * Support large scale deployments - 10s of millions of repositories
+  * Support performance large scale deployments (using indexes or not) to be able to execute the 90% of queries under 30mins of execution.
+  * Improve the error reporting to minimize the number of issues open.
+* Significantly improve the engineering team workflow
+  * Deliver products on time, without commit other teams.
```

Comment:
This is not *objective*, what does it mean to deliver products on time exactly?
Is your goal to always deliver on time? Are you tracking when you don/t?

Reply:
I see the goal of deliver the project on time and archive this OKRs, is kind of meta, but I think is a objetive thing.